packages/firebase_admob/android/src/main/java/io/flutter/plugins/firebaseadmob/FirebaseAdMobPlugin.java

```diff
@@ -44,7 +45,47 @@ private void callInitialize(MethodCall call, Result result) {
     result.success(Boolean.TRUE);
   }
 
-  private void callLoadAd(MobileAd ad, MethodCall call, Result result) {
+  private void callLoadBannerAd(int id, Activity activity, MethodChannel channel, MethodCall call, Result result) {
+    String adUnitId = call.argument("adUnitId");
+    if (adUnitId == null || adUnitId.isEmpty()) {
+      result.error("no_unit_id", "a non-empty adUnitId was not provided for ad id=" + id, null);
```

Comment:
This wording is a bit confusing. Would you be open to changing this to something like `a null or empty adUnitId...`?

Reply:
Done.